app/app.js

```diff
@@ -36,7 +39,7 @@ window.run = function() {
                 };
             });
 
-        var setupIosLayout = function(counterBadgeControllerPromise) {
+        var tabBarLayout = function(counterBadgeControllerPromise) {
```

Comment:
maybe this should be named with a verb instead of a noun since it is a function. Noun implies variable. Maybe `createTabBarLayout`?